run/idp-sql/Dockerfile

```diff
@@ -6,6 +6,9 @@
 # https://hub.docker.com/_/python
 FROM python:3.9
 
+# Send stdout/stderr out, do not buffer.
+ENV PYTHONUNBUFFERED 1
```

Comment:
I'm wondering if this is the best method for flushing logs

Reply:
"Force the stdout and stderr streams to be unbuffered." https://docs.python.org/3/using/cmdline.html#envvar-PYTHONUNBUFFERED

You can also use `import sys; sys.stdout.flush()` to manually force a flush. 

For this example, should I be including multiple settings to ensure logs are flushed, connections are closed, etc, even though they may not have any effect?

Reply:
> For this example, should I be including multiple settings to ensure logs are flushed, connections are closed, etc, even though they may not have any effect?

No, just the main way. A lot of my questions are figuring out what is idiomatic to Python versus what I've seen elsewhere. I assume `PYTHONUNBUFFERED` is the more idiomatic/common way?

Reply:
I believe it is, as it's the method we've used to ensure timely logs getting received by Cloud Logging when working with Cloud Functions.